public/500.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
If it's just an API, you can choose to remove those (after all, you're gonna respond with a JSON)